src/App.jsx

```diff
@@ -10,34 +12,39 @@ import styles from './App.css';
 function RGB() {
   // 🚨 Get values for r, g, and b from params
   // 🚨 Uncomment next lines after importing
-  // return (
-  // <div
-  //   className={styles.fill}
-  //   style={{ background: `rgb(${r}, ${g}, ${b})` }}
-  // >
-  //   <p>{`rgb(${r},${g},${b})`}</p>
-  // </div>
-  // );
+  const { r, g, b } = useParams();
+  return (
+    <div
+      className={styles.fill}
+      style={{ background: `rgb(${r}, ${g}, ${b})` }}
+    >
+      <p>{`rgb(${r},${g},${b})`}</p>
+    </div>
+  );
 }
 
 function ScreenColor() {
-  return <div>{/* Create Route Inside Switch */}</div>;
+  return <div></div>;
 }
 
 export default function App() {
   return (
     <Router>
       <header>
-        <Route exact path="/">
-          <Redirect to="/rgb/192/192/192" />
-        </Route>
         <ul>
           <Link to="/rgb/192/192/192">Silver</Link>
           <Link to="/rgb/220/20/60">Crimson</Link>
           <Link to="/rgb/147/112/219">Purple</Link>
         </ul>
+        <Switch>
+          <Route exact path="/">
+            <Redirect to="/rgb/192/192/192" />
+          </Route>
+          <Route path={'/rgb/:r/:g/:b'}>
+            <RGB />
+          </Route>
```

Comment:
You will want to render this within the `ScreenColor `component since it's rendered in `App`. The page will continue to work but refactoring this would be best.